applications/nrf_desktop/Kconfig.ble

```diff
@@ -75,6 +75,26 @@ config BT_GATT_CLIENT
 
 endif
 
+config DESKTOP_BT_CTLR
+	bool
+	default y if SOC_SERIES_NRF52X
```

Comment:
Relying on SoC series may be a bit tricky I think (e.g. what about nRF51 series)?
Maybe we could avoid introducing this option and simply overlay defaults if BT_CTLR is enabled? Boards seem to automatically enable the BT_CTLR if BT is enabled.

Reply:
I would like to use this configuration for the HCI Rpmsg at some point.

Do we support the nRF51 series? In this case, I can add it. Plus, it is always possible to enable this option in the prj.conf file.

Reply:
Discussed offline. We cannot modify configuration for child image like that, so we can only apply this if Bluetooth controller is built as part of application image. We should not rely on SoC series here, so we can just apply the overlays if BT_CTLR is enabled,